exercises/queen-attack/description.md

```diff
@@ -8,6 +8,7 @@ A chessboard can be represented by an 8 by 8 array.
 
 So if you are told the white queen is at `c5` (zero-indexed at column 2, row 3) and the black queen at `f2` (zero-indexed at column 5, row 6), then you know that the set-up is like so:
 
+###### B and W are the queens, standing for Black and White. This is not relevant info to the puzzle.
```

Comment:
This would make more sense as an [admonition](https://exercism.org/docs/building/markdown/markdown#h-special-blocks-sometimes-called-admonitions).

Reply:
Yes, this should not be a heading. I think having the content here is a good idea by @T-O-R-U-S , but I would wrap it in:

`````markdown
~~~~exercism/note
B and W are the queens, standing for Black and White.
This is not relevant info to the puzzle.
~~~~
`````

Reply:
@T-O-R-U-S I see that you already added the proposed note, but also make sure to remove this heading.